### PR TITLE
Dx 944 cobadge card

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: payments/card/other-features.md [hunk 1]
```diff
@@ -70,6 +70,25 @@ document is described in the following section.
...
+Read more about this EU legislation at [Dankort][dankort-eu].
+
+As a merchant you are able to set a priority selection of payment application by
+contacting [SwedbankPay Support][swedbankpay-support]. The end user will always
+be able to override this priority selection on the payment page.
+
+If you want more information about Co-badge Card Choice for Dankort users please contact
+[SwedankPay Support][swedbankpay-support]. The example below shows the
+payment window where there payer can select between Dankort or Visa before completing
+the payment.
```
Comment (asbjornu) [inline payments/card/other-features.md:88]:
```suggestion
Due to EU regulations from 2016-06-09 regarding cards that have more than
one payment application, we have developed support for the end users of Dankort
to be able to choose their preferred payment application on the Swedbank Pay
payment page. Read more about this EU legislation at [Dankort][dankort-eu].

As a merchant you are able to set a priority selection of payment application by
contacting [SwedbankPay Support][swedbankpay-support]. The end user will always
be able to override this priority selection on the payment page.

If you want more information about Co-badge Card Choice for Dankort users please
contact [SwedankPay Support][swedbankpay-support]. The example below shows the
payment window where there payer can choose between Dankort or Visa before
completing the payment.
```

File: payments/card/other-features.md [hunk 2]
```diff
@@ -70,6 +70,25 @@ document is described in the following section.
...
+
+![card-badge][card-badge]{:height="500px" width="425px"}
```
Comment (asbjornu) [inline payments/card/other-features.md:90]:
```suggestion
![Co-badge Dankort cards with option to choose between Dankort and Visa before paying][card-badge]{:height="500px" width="425px"}
```

File: payments/card/other-features.md [hunk 3]
```diff
@@ -196,6 +215,8 @@ payment"  %}
 [cancel]: /payments/card/after-payment#cancellations
 [capture]: /payments/card/capture
 [callback]: /payments/card/other-features#callback
+[card-badge]: /assets/img/card-badge.png
+[dankort-eu]:  https://www.dankort.dk/Pages/Dankort-eller-Visa.aspx
```
Comment (NullableInt) [inline payments/card/other-features.md:219]:
```suggestion
[dankort-eu]: https://www.dankort.dk/Pages/Dankort-eller-Visa.aspx
```

File: payments/card/other-features.md [hunk 4]
```diff
@@ -70,6 +70,24 @@ document is described in the following section.
 
 {% include prices.md %}
 
+## Co-badge Card Choice for Dankort
+
+Due to EU regulations from 2016-06-09 regarding cards that have more than
```
Comment (asbjornu) [inline payments/card/other-features.md:75]: Perhaps add the link you found as a reference from here?

```suggestion
Due to [EU regulations from 2016-06-09][eu-regulations] regarding cards that have more than
```

**Pull request merged**